test/lib/ufe/testUfePythonImport.py

```diff
@@ -26,13 +26,19 @@ class UfePythonImportTestCase(unittest.TestCase):
 
     def testImportModule(self):
         from mayaUsd import ufe as mayaUsdUfe
+        import ufe
 
         # Test calling a function that depends on USD. This exercises the
         # script module loader registry function and ensures that loading the
         # ufe library also loaded its dependencies (i.e. the core USD
         # libraries). We test the type name as a string to ensure that we're
         # not causing USD libraries to be loaded any other way.
-        invalidPrim = mayaUsdUfe.getPrimFromRawItem(0)
+        if ufe.VersionInfo.getMajorVersion() >= 2:
+            # raw item only introduced in UFE v2
+            invalidPrim = mayaUsdUfe.getPrimFromRawItem(0)
+        else:
+            # In UFE v2 if the Nb segments == 1, returns invalid prim
+            invalidPrim = mayaUsdUfe.ufePathToPrim('a')
```

Comment:
This test is avail in UFE v1, but it was using a ufe wrapper method that I've made only avail in UFE v2. So for the v1 case use a different wrapper method. We cannot just use ufePathToPrim for both UFE versions, since in UFE v2 ufePathToPrim uses different code that will assert with an invalid input path.